modules/recon/domains-contacts/metacrawler.py

```diff
@@ -2,7 +2,7 @@
 from recon.mixins.search import GoogleWebMixin
 from PyPDF2 import PdfFileReader
 from PyPDF2.utils import PdfReadError
-from StringIO import StringIO
```

Comment:
One thing I ran into while testing was that many places where I was using StringIO on py2 now required the inverse of BytesIO on the other side of the operation in py3. Something to keep an eye out for.